src/Diagnostics.Scripts/Models/EntityMetadata.cs

```diff
@@ -6,16 +6,18 @@ public sealed class EntityMetadata
 
         public string ScriptText;
         public string Metadata;
+        public string LastModifiedMarker;
```

Comment:
lets call it Sha here in the EntityMetadata Model instead of LastModifiedMarker?